pipeline_config.py

```diff
@@ -98,9 +98,103 @@
         },
+    'thresholder': {'threshold': params.threshold},
+    'watershed': {},
+    'dropper': {'min_size': params.min_nuclei_size},
+    'postprocessor':{}
+},
+ 'unet_mask': {
+        'architecture_config': {'model_params': {'n_filters': params.n_filters,
+                                                 'conv_kernel': params.conv_kernel,
+                                                 'pool_kernel': params.pool_kernel,
+                                                 'pool_stride': params.pool_stride,
+                                                 'repeat_blocks': params.repeat_blocks,
+                                                 'batch_norm': params.use_batch_norm,
+                                                 'dropout': params.dropout_conv,
+                                                 'in_channels': params.image_channels,
+                                                 'nr_outputs': params.nr_unet_outputs
+                                                 },
+                                'optimizer_params': {'lr': params.lr,
+                                                     },
+                                'regularizer_params': {'regularize': True,
+                                                       'weight_decay_conv2d': params.l2_reg_conv,
+                                                       },
+                                'weights_init': {'function': 'xavier',
+                                                 },
+                                'loss_weights': {'mask': params.mask_mask,
+                                                 'contour': params.mask_contour,
+                                                 'contour_touching': params.mask_contour_touching,
+                                                 'center': params.mask_center,
+                                                 },
+                                },
+        'training_config': {'epochs': params.epochs_nr,
+                            'shuffle': True,
+                            'batch_size': params.batch_size_train,
+                            },
+        'callbacks_config': {
+            'model_checkpoint': {
+                'filepath': os.path.join(GLOBAL_CONFIG['exp_root'], 'checkpoints_mask', 'network', 'best.torch'),
+                'epoch_every': 1},
+            'lr_scheduler': {'gamma': params.gamma,
+                             'epoch_every': 1},
+            'training_monitor': {'batch_every': 0,
+                                 'epoch_every': 1},
+            'experiment_timing': {'batch_every': 0,
+                                  'epoch_every': 1},
+            'validation_monitor': {'epoch_every': 1},
+            'neptune_monitor': {    'model_name': 'unet_mask',
+                                    'image_nr': 4,
+                                'image_resize': 0.2},
+            'early_stopping': {'patience': params.patience},
+        },
+    },
+    'unet_contour': {
```

Comment:
all this should be defined in the neptune.yaml with a seperate prefix. For example 'unet_contour_n_filters'